cloudbuild.yaml

```diff
@@ -37,6 +37,17 @@ steps:
     - 'REPO_NAME=${REPO_NAME}'
     - 'COMMIT_SHA=${COMMIT_SHA}'
 
+# Run authorization logic tests 
+- id: 'auth_logic_tests'
+  waitFor: ['build-image']
+  name: '${_DOCKER_IMAGE}'
+  entrypoint: 'bash'
+  args: [
+    'cd rust/tools/authorization-logic &&',
```

Comment:
It is failing with the following error: 

`bash: cd rust/tools/authorization-logic &&: No such file or directory`

You might want to use manifest-path argument: https://doc.rust-lang.org/cargo/commands/cargo-test.html. Change args as follows:
```
args: ['cargo', 'test', '--manifest-path=rust/tools/authorization-logic']
```

Another alternative is to create a `gcb/cargo_test.sh` that changes to the directory and runs `cargo test`. This will also help us add some better status messages later if needed. If you go down the route, then this cloudbuild.yaml rule would change as follows:

```
- id: 'auth_logic_tests'
  waitFor: ['build-image']
  name: '${_DOCKER_IMAGE}'
  entrypoint: 'bash'
  args: [ '.gcb/cargo-test.sh']
  ]

Reply:
Thanks. Sorry I had to step away since it was evening in my time zone.

I ran into [this issue](https://stackoverflow.com/questions/61055324/docker-cannot-execute-binary-file), so it looks like it also needed `-l -c`. At least when I launch the gcb job manually from the CLI, it works. So it should be OK now.